CHANGES.rst

```diff
@@ -10,6 +10,24 @@ esa/hubble
 - Module added to query eHST TAP based on a set of specific criteria and
   asynchronous jobs are now supported. [#1723]
 
+SVO FPS
+^^^^^^^
+
+- Module added to access the Spanish Virtual Observatory Filter Profile List [#1498]
+
+Splatalogue
+^^^^^^^^^^^
+
+- The Splatalogue ID querying is now properly cached in the `astropy` cache
+  directory (Issue [#423]) The scraping function has also been updated to reflect
+  the Splatalogue webpage. [#1772]
+
+UKIDSS
+^^^^^^
+
+- Updated to ``UKIDSSDR11PLUS`` as the default version [#1767]
+
+
```

Comment:
somehow these changelog entries have been added - they're not part of this PR.  Maybe they got unintentionally moved here during the squash?

Reply:
It seems like at least some if not all of the actual changes associated with these changelog entries have also been added.

Reply:
I have no idea how they got in and where they are coming from. For me, re-base is always a nightmare. And I'm not sure about the benefits.

Reply:
Can someone tell me why it thinks this PR added those changes when they are already in master?

Reply:
are they maybe duplicated in different sections here?

Reply:
`github` flagged a conflict with `master`. Fixed it and the extra files are now gone. Please double-check.

Reply:
Can someone assign it to the next milestone to make sure I'm not missing anything? Thanks

Reply:
At the time of tagged release the changelog gets a restructure, I think that has tricked your git to think you have all these as new additions. Anyway, it's all been cleared out now.